config/locales/en.yml

```diff
@@ -187,8 +187,8 @@ en:
             select_organisations: You can change this later from your dashboard using the filters
           labels:
             select_organisations: Select which schools' job listings you would like to see
-            select_organisations_html: >-
-              Select which schools' job listings you would like to view (<span class='text-red'>Required</span>)
+            select_organisations: >-
```

Comment:
This has now moved to the new file publishers.yml

Reply:
yes i saw that

Reply:
I approved your PR that moved the file